cloud-tasks/app/index.html

```diff
@@ -0,0 +1,97 @@
+<head>
+  <meta charset="UTF-8">
+  <title>Send a Postcard</title>
+  <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <style>
+    body {
+      text-align: center;
+    }
+    .mdc-card {
+      width: 800px;
+    }
+    .center .mdc-card {
+      margin: auto;
+      text-align: center;
+    }
+    .mdc-text-field {
+      margin: 1em;
+    }
+  </style>
+</head>
+<body class="mdc-typography">
+
+  <div class="center">
+    <h1 class="mdc-typography--headline2"> Send a E-Postcard! </h1>
+    <div class="mdc-card mdc-card--outlined">
+      <!-- Postcard form -->
+      <form action="/send-email" method="post">
+        <div class="mdc-layout-grid">
+          <div class="mdc-layout-grid__inner">
+            <!-- Message textarea -->
+            <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
+              <h3>Your message:</h3>
+              <div class="mdc-text-field mdc-text-field--textarea mdc-text-field--no-label">
+                <textarea id="textarea" class="mdc-text-field__input"
+                  rows="8" cols="40" name="message" aria-label="Label"></textarea>
+                <div class="mdc-notched-outline">
+                  <div class="mdc-notched-outline__leading"></div>
+                  <div class="mdc-notched-outline__trailing"></div>
+                </div>
+              </div>
+            </div>
+            <!-- Sender/Receiver Emails -->
+            <div class=" mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
+              <div class="mdc-text-field mdc-text-field--no-label">
+                <input type="text" class="mdc-text-field__input" name="to_email"
+                  placeholder="Recipient Email" aria-label="Label" required>
+              </div>
+
+              <div class="mdc-text-field mdc-text-field--no-label">
+                <input type="text" class="mdc-text-field__input" name="from_email"
+                  placeholder="Your Email" aria-label="Label" required>
+              </div>
+              <div>
+                <h3>Schedule your postcard:</h3>
+                <input id="date-input" type="date" name="date">
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="mdc-card__actions">
+          <div class="mdc-card__action-buttons">
+            <button class="mdc-button mdc-card__action mdc-card__action--button">
+              <span class="mdc-button__label">Send</span>
+              <i class="material-icons mdc-button__icon" aria-hidden="true">send</i>
+            </button>
+          </div>
+        </div>
+      </form>
+
+    </div>
+  </div>
+  <script>
+  let now = new Date();
+
+  let maxDate = new Date(now);
+  maxDate.setDate(now.getDate() + 30)
+
+  document.getElementById('date-input')
+    .setAttribute('min', now.toISOString().substring(0,10))
```

Comment:
Nit: `0,10` should be `0, 10`.
This is substring operation is a bit hacky, but I guess there's not much of a better solution without a lot of code.

Reply:
ack.